decidim-core/app/packs/src/decidim/editor/extensions/indent/index.js

```diff
@@ -13,7 +13,7 @@ const allowedNodeTypes = ["heading", "paragraph"];
 const allowedNodeActive = (editor) => allowedNodeTypes.some((type) => editor.isActive(type));
 
 /**
- * Finds the closest allowed type node from the given position. Traversese the
+ * Finds the closest allowed type node from the given position. Traverse the
```

Comment:
I would say that it is better to have `Traverses` as in `Traverses ... to search through`

```suggestion
 * Finds the closest allowed type node from the given position. Traverses the
```

Reply:
Sure. I was very much on the fence on it